src/README.md

```diff
@@ -16,10 +16,13 @@ To build protobuf from source, the following tools are needed:
   * automake
   * libtool
   * curl (used to download gmock)
+  * make
+  * clang (gcc)
```

Comment:
Switch the place of clang and gcc? I'm under the impression that gcc is still the default compiler coming with Linux distributions. And make it g++ because we require a C++ compiler?